Detections/MultipleDataSources/IridiumIOCs.yaml

```diff
@@ -0,0 +1,134 @@
+id: 7ee72a9e-2e54-459c-bc8a-8c08a6532a63
+ 
+name: Known IRIDIUM IP
+ 
+description: |
+ 
+  'IRIDIUM command and control IP. 
+ 
```

Comment:
Do you realy need here the emapty lines?
It will display like this in Sentinel UI.

Reply:
Thanks. Removed Empty lines from the query.